packages/runtime/runtime-utils/src/test/utils.spec.ts

```diff
@@ -16,7 +16,7 @@ describe("Utils", () => {
 	});
 
 	it("encodeCompactIdToString() has base of 64 (sort of)", () => {
-		const chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ[abcdefghijklmnopqrstuvwxyz{01234567890";
+		const chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ(abcdefghijklmnopqrstuvwxyz)01234567890";
```

Comment:
You could add a single test case that `encodeURIComponent(chars) === chars`

Reply:
nit: There are some other characters that are safe to encode - `- _ . ! ~ * '`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent#description